src/transformStateWithClones.js

```diff
@@ -66,6 +66,29 @@
  */
 function transformStateWithClones(state, transforms) {
   // write code here
+  const stateArr = [];
+  let newState = Object.assign({}, state);
```

Comment:
I think it is better to use `{...state}` in this situation

Reply:
Agree with Marina. Destructuring is shorter and is easy to read.